solutions/kubernetes/ceph-binary-build.sh

```diff
@@ -176,7 +176,7 @@ function ceph_build() {
 
                     pushd rpmbuild
                         add_common_separator "Start Build"
-                        rpmbuild --define "_topdir `pwd`" -ba SPECS/ceph.spec
+                        rpmbuild --define "_topdir $(pwd)" -ba SPECS/ceph.spec
```

Comment:
Please check that the scripts work with this change

Reply:
Reverted it back